### ISSUE TITLE
[ADE 508 Audit]: 508 Audit Report for 44578 MyHealTheVet (MHV) on VA.GOV: 508 Audit Report Findings Support -Defect 9 | High

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#113317]
> [!NOTE]
> **QUICK SUMMARY:** _Child ticket of https://github.com/department-of-veterans-affairs/va.gov-team/issues/113312 ._

### Defect 9 | High | Created:
_WCAG 4.1.1 Parsing | In content implemented using markup languages, elements have complete start and end tags, elements are nested according to their specifications, elements do not contain duplicate attributes, and any IDs are unique, except where the specifications allow these features._

Throughout the product, the names for some elements are read twice by assistive technology when reading with the arrow keys. For example, on the Medications | Veterans Affairs screen (Filter list), when reading with the arrow keys, “All medications (341) All medications in your VA medical records radio button checked 1 of 5 All medications (341)All medications in your VA medical records” is read. Similarly, on the Refill prescriptions - Medications | Veterans Affairs screen (Start a refill request), “Select all 8 refills check box not checked Select all 8 refills” is read. (New: 06/27/2025)

On the Medications | Veterans Affairs screen, after expanding What to know before you print or download, the content after If you’re on a public or shared computer, cannot be read by assistive technology. (New: 06/27/2025)

### Ticket Updates
- [ ] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [ ] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._

**Comment by coforma-jamie** [comment]
**Update:** @jeana-adhoc did some testing as well. 

Between the two of us, we've determined that the Additional info component bug **is** an issue in:

- JAWS 2023.2402.1 (February 2024 release)
- JAWS 2024-2409.2 (September 2024 release)
- JAWS 2025.2412.50 (December 2024 release)

But it's **not** an issue in:

- JAWS 2025-2505.43 (May 2025 release)
- Mac OS VO
- iOS VO
- NVDA 2025.1.1 (June 2025 release)

This raises an important question - which versions of JAWS (or any assistive tech) are we supporting? The latest version? The previous 5 versions? Further back still? 

@artsymartha68, I assume this is a question for you. 😄 Whatever you decide, we'll need to communicate that to the Collab Cycle folks as well so they know what versions of JAWS to test with.

**Comment by coforma-jamie** [comment]
Summarized the above comments as a comment on this VADS ticket, per @jeana-adhoc: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4309

**Comment by coforma-jamie** [comment]
[Comment in parent ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/113312#issuecomment-3045571482)
